front/src/components/ViewMockup.js

```diff
@@ -49,19 +49,66 @@ class ViewMockup extends Component {
         window.location.href = "/mockupsurvey/add/" + this.state.mockup.id
     };
 
+    startButton() {
+        function startRecording(stream) {
```

Comment:
could this function be outside of the `startButton()`? that would make it shorter